tests/009-test-change-lv-data-size.sh

```diff
@@ -0,0 +1,80 @@
+source $SRCDIR/libtest.sh
+
+# Test that the DATA_SIZE option for d-s-s actually 
+# changes the size of the lv created from the volume group.
+
+test_change_lv_data_size() {
+  local devs=$TEST_DEVS
+  local test_status
+  local testname=`basename "$0"`
+  local vg_name
+  local vg_name="dss-test-foo"
+  local pe_total
+  local pe_free
+  local pe_used_percentage
+  local data_size=70
+  
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  for vg in $(vgs --noheadings -o vg_name); do
+    if [ "$vg" == "$vg_name" ]; then
+      echo "ERROR: $testname: Volume group $vg_name already exists."
+      return 1
+    fi
+  done
+
+  # Create config file with Docker storage options
+  clean_config_files
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+DEVS="$devs"
+VG=$vg_name
+DATA_SIZE=$data_size%FREE
```

Comment:
Ok, here you are checking for 70%FREE. I think we can get rid of previous test. 

Don't like the idea of tests failing if somebody is changing defaults.